docs/api.html

```diff
@@ -8015,6 +8023,46 @@
         "tags" : [ "/api/ldap" ]
       }
     },
+    "/api/logger/adjust" : {
+      "post" : {
+        "summary" : "POST /api/logger/adjust",
+        "description" : "Get the namespaces of preset `id`.",
+        "parameters" : [ ],
+        "requestBody" : {
+          "content" : {
+            "application/json" : {
+              "schema" : {
+                "type" : "object",
+                "properties" : {
+                  "duration" : {
+                    "type" : "integer"
+                  },
+                  "duration_unit" : {
```

Comment:
Are `_` characters the preferred separators? I.e., should it be snake_case or kebab-case?

Reply:
Most endpoints follow `snake_case`.
If we could rename `undo-task` to `undo_task` that'd be great.

Reply:
Sorry, I didn't mean to return that. Actually, I didn't mean to return anything at all. Is the plan useful in the FE, do you think?

Reply:
It's not useful in FE

Reply:
Removed the return value.